backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -750,6 +724,8 @@ public Project deepCopyProjectForUser(String srcProjectID, UserInfo newOwner)
       LOGGER.debug("Mark Projects as deleted : {}, count : {}", allowedProjectIds, updatedCount);
       transaction.commit();
       LOGGER.debug("Project deleted successfully");
+
+      roleService.deleteProjectResources(allowedProjectIds);
```

Comment:
We should let the GC process handle this during soft delete.

Reply:
+:+1: